resources/js/components/FormField.vue

```diff
@@ -22,7 +22,11 @@ export default {
         registerDependencyWatchers(root) {
             root.$children.forEach(component => {
                 if (this.componentIsDependency(component)) {
-                    if (component.selectedResourceId !== undefined) {
+                    if (component.selectedResource !== undefined) {
+                        //Searchable BelongsTo field
+                        component.$watch('selectedResource.value', this.dependencyWatcher, {immediate: true});
+                        this.dependencyWatcher(component.selectedResource.value);
```

Comment:
Not sure about this line, it also causes an error in the log.
This might be
`this.dependencyWatcher(component.selectedResourceId);`
It works even without this line but I guess it's for functionality I'm not currently using.